web_external/templates/body/studyResultsPage.jade

```diff
@@ -0,0 +1,26 @@
+.isic-form-page.isic-study-results-container
+  h2.isic-page-title Annotation Study Results
+
+  .isic-form-container
+    .isic-form-group.isic-study-results-form-group
+
+      label.control-label(for="isic-study-results-study") Study
+      select#isic-study-results-select-study-container.form-control.input-sm
+        option(selected disabled value='')
+          | Select...
+
+      label.control-label(for="isic-study-results-image") Image
+      select#isic-study-results-select-image-container.form-control.input-sm(disabled)
+        option(selected disabled)
+          | Select...
+
+      label.control-label(for="isic-study-results-user") User
+      select#isic-study-results-select-user-container.form-control.input-sm(disabled)
+        option(selected disabled)
+          | Select...
+
+    .isic-form-group.isic-study-results-form-group
+       #isic-study-results-global-features-container
+
+     .isic-form-group.isic-study-results-form-group
```

Comment:
Thanks. I'm trying a different editor and need to work out its configuration to make some things more automatic.